src/zrc/datamodel/models.py

```diff
@@ -172,17 +181,33 @@ class Zaak(APIMixin, models.Model):
 
     selectielijstklasse = models.URLField(
         _("selectielijstklasse"), blank=True,
-        help_text=_("URL-referentie naar de categorie in de gehanteerde "
-                    "'Selectielijst Archiefbescheiden' die, gezien het zaaktype "
-                    "en het resultaattype van de zaak, bepalend is voor het "
-                    "archiefregime van de zaak.")
+        help_text=_("URL-referentie naar de categorie in de gehanteerde 'Selectielijst Archiefbescheiden' die, gezien "
+                    "het zaaktype en het resultaattype van de zaak, bepalend is voor het archiefregime van de zaak.")
     )
 
     relevante_andere_zaken = ArrayField(
         models.URLField(_("URL naar andere zaak"), max_length=255),
         blank=True, default=list
     )
 
+    # Archiving
+    archiefnominatie = models.CharField(
+        _("archiefnominatie"), max_length=40, null=True, blank=True,
```

Comment:
is there a reason for the `null=True`?

Reply:
It's an optional field with an enumeration. I think `null` is the better choice than an empty string.

Reply:
I don't think that translates well to the admin though, you could have a `null` value, save the object in the admin and then it'll be an empty string *I think*. API-wise I agree with the reasoning.

DRF doesn't seem to handle it without a struggle though: https://github.com/encode/django-rest-framework/issues/2623